src/pytest_elasticsearch/plugin.py

```diff
@@ -17,18 +17,32 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with pytest-elasticsearch.  If not, see <http://www.gnu.org/licenses/>.
 """Pytest-elasticsearch py.test's plugin configuration."""
+import pytest
+
 from pytest_elasticsearch import factories
 
 
 def pytest_addoption(parser):
     """Add plugin's configuration options."""
+    parser.addini(
+        name='elasticsearch_logsdir',
+        help='',
+        default='/tmp',
+    )
+
     parser.addoption(
         '--elasticsearch-logsdir',
         action='store',
-        default='/tmp',
         metavar='path',
         dest='logsdir',
     )
 
+
+def elasticsearch_config():
```

Comment:
If it's test related it shouldn't be here. If that's supposed to be used by fixtures, then it maybe could be a fixture itself?